cmake/Windows/NSIS.template.in

```diff
@@ -486,10 +528,16 @@ Section "-PathInst" SEC_PathRegister
     done:
       FindClose $0
 
-    #create OPENMS_DATA_PATH environment variable (for shared xml files etc)
-    ; set variable
-    WriteRegExpandStr ${env_hklm} "OPENMS_DATA_PATH" "$INSTDIR\share\OpenMS"
-    ; make sure windows knows about the change
+    # Create OPENMS_DATA_PATH environment variable (for shared xml files etc)
+
+    # Set variable
+    ${If} $MultiUser.InstallMode == "AllUsers"

```

Comment:
is this the If from logiclib or build in?

Reply:
I think logiclib. As mentioned they are included by the Multiuser plugin.